match/match.py

```diff
@@ -130,6 +131,79 @@ async def match(self, ctx, *args):
                 )
         await ctx.message.delete()
 
+    @commands.command(aliases=['lobbyup', 'up'])
+    @commands.guild_only()
+    async def lobbyready(self, ctx):
+        """Informs players of the opposing team that the private match lobby is ready and joinable."""
+        match_day = await self._match_day(ctx)
+        teams = await self.team_manager.teams_for_user(ctx, user)
+        
+        if not (match_day and teams):
+            return
+
+        team_name = teams[0]
+        match_data = await self.get_match_from_day_team(ctx, match_day, team_name)
```

Comment:
What happens if there is no match data returned here? It doesn't look like we're handling that situation gracefully so I imagine it will thrown a null error somewhere along the way in the next few lines.

Reply:
It does a ctx reply:
`:x: No match found.`
![image](https://user-images.githubusercontent.com/16944592/118588878-47de4780-b76d-11eb-8db2-92c20b624282.png)